library/src/test/java/bio/terra/landingzone/library/landingzones/TestArmResourcesFactory.java

```diff
@@ -84,6 +84,7 @@ public static ResourceGroup createTestResourceGroup(AzureResourceManager azureRe
         .resourceGroups()
         .define("rg" + resourceGroupId)
         .withRegion(Region.US_SOUTH_CENTRAL)
+        .withTag("PURPOSE", "LANDING_ZONE_SERVICE_INTEGRATION_TESTING")
```

Comment:
I want an easier way to determine if an RG is deletable besides inferring from an opaque "rg<some_uuid>" naming scheme.